bundler/Rakefile

```diff
@@ -81,10 +81,10 @@ namespace :man do
       Spec::Rubygems.gem_require("ronn")
     rescue Gem::LoadError => e
       desc "Build the man pages"
-      task(:build) { abort "We couln't activate ronn (#{e.requirement}). Try `gem install ronn:'#{e.requirement}'` to be able to build the help pages" }
+      task(:build) { abort "We couldn't activate ronn (#{e.requirement}). Try `gem install ronn:'#{e.requirement}'` to be able to build the help pages" }
```

Comment:
While working on getting set up locally, I just stumbled on and fixed this typo.

Reply:
Thanks! In general, I try to split unrelated changes to separate commits, but it's no big deal :)